app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -79,7 +79,7 @@ fun ButtonBar() {
 @Composable
 fun Menu() {
     val items = listOf("A", "B", "C", "D", "E", "F")
-    var countryName : String by remember { mutableStateOf("Menu Categories") }
+    val menuName: String by remember { mutableStateOf("Menu Categories") }
```

Comment:
Renamed this variable to fit better with the theme of the project. Immutable data should be declared as a val instead of a var. Immutability best practices can be found here. https://kotlinlang.org/docs/coding-conventions.html#immutability